### ISSUE TITLE
Smee.io not receiving webhook payloads

### ISSUE THREAD

**Issue by praveendvd** [probot/smee-client#132]
I created a channel and tried editing the github repository. But events are not received by smee.io channel

**Comment by praveendvd** [comment]
On further analysis, i have to do two pushes then only the event is getting send to client

**Comment by stefankip** [comment]
Yes we're also having issues with github and smee.io webhooks. So this isn't a github issue?

**Comment by praveendvd** [comment]
> Yes we're also having issues with github and smee.io webhooks. So this isn't a github issue?

No , i verified that github web-hooks are generated , by using  https://webhook.site/
